pages/common/csvsql.md

```diff
@@ -0,0 +1,17 @@
+# csvsql
+
+> Generate SQL statements for a CSV file or execute those statements directly on a database.
+> Included in csvkit.
+> More information: <https://csvkit.readthedocs.io/en/latest/scripts/csvsql.html>.
+
+- Generate a `CREATE TABLE` SQL statement for a CSV file:
+
+`csvsql {{data.csv}}`
```

Comment:
Perhaps we could adjust this to be more consistent with other pages? Same goes for other instances of `data.csv`.
```suggestion
`csvsql {{path/to/data.csv}}`
```